platform/cog-platform-drm.c

```diff
@@ -236,6 +279,180 @@ init_egl ()
 }
 
 
+static void
+input_handle_key_event (struct libinput_event_keyboard *key_event)
+{
+    struct wpe_input_xkb_context *default_context = wpe_input_xkb_context_get_default ();
+    struct xkb_state *state = wpe_input_xkb_context_get_state (default_context);
+
+    uint32_t key = libinput_event_keyboard_get_key (key_event) + 8;
```

Comment:
Why do we need a `+ 8` here? I am sure it's needed to make things work, but it would be good to add a comment explaining why, or a `TODO: Fix $FOO` if this is a temporary hack to get things working without delving into more complexity right now.

Reply:
Wow, interesting, thanks for adding the comment—it will prevent us from thinking that it may be a good idea to remove it when going over this code at some point in the future.